editor/components/provider/index.js

```diff
@@ -52,6 +52,12 @@ class EditorProvider extends Component {
 		// Assume that we don't need to initialize in the case of an error recovery.
 		if ( ! props.recovery ) {
 			this.store.dispatch( setupEditor( props.post, this.settings ) );
+
+			// @todo only show the autosave alert when there exists an autosave newer than the post
```

Comment:
Not sure I understand this `todo` is this a leftover? since we already show the alert.

Reply:
Yes, leftover

Reply:
removed